independent-projects/arc/runtime/src/main/java/io/quarkus/arc/impl/ArcContainerImpl.java

```diff
@@ -199,7 +200,7 @@ public InjectableContext getActiveContext(Class<? extends Annotation> scopeType)
                 if (context.isActive()) {
                     if (selected != null) {
                         throw new IllegalArgumentException(
-                                "More than one context object for the given scope: " + selected + " " + context);
+                                "More than one context object for the given scope: " + selected + ' ' + context);
```

Comment:
This is suggested by the IDE. I have the intuition that it improves the performance, since we are using a primitive instead of a String. But I never did a benchmark and couldn't find any recent article that talks about this. I agree that this would be a premature optimization, but since it doesn't make less readable, I would trust on the IDE. WDYT?

Reply:
I'd rather stay consistent. We don't need any perf improvements here..